schemas/accelerator.sql

```diff
@@ -0,0 +1,397 @@
+-- This schema adds the tables necessary for Accelerator accounting as a
+-- separate record as part of the wider Cloud Accounting system.
```

Comment:
It's not dependant on `cloud.sql` though, is it?

Reply:
It is

Reply:
Just had a look to confirm - the only coupling is that one of the functions (SiteLookup()) is used in ReplaceAcceleratorRecord and that function is contained within cloud.sql.  I think the best approach here would be to set an issue to decouple all the miscellaneous lookup functions for categories and elements that are the same into a utils.sql, then run that inside accelerator.sql and/or cloud.sql as there's currently some duplication.  It shouldn't be a hard fix but I don't have time to figure it out